### PR TITLE
Added Spanish translation

### PR DESCRIPTION
**Pull request by AlejandroC90**
Spanish translation of gnome-twitch, check if anything is wrong.
I couldnt check the "Player" string context, so I used the other translation, somehow can't load top channels.
image

### CODE REVIEW

File: po/es.po [hunk 1]
```diff
@@ -0,0 +1,248 @@
+# SOME DESCRIPTIVE TITLE.
```
Comment (dengelt) [inline po/es.po]: Please have a look at the other *.po files to see how the header should look.

File: po/es.po [hunk 2]
```diff
@@ -0,0 +1,248 @@
...
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the gnome-twitch package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
+# Holman Calderón <halecalderon@gmail.com>, 2016.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: gnome-twitch 0.3.0\n"
```
Comment (dengelt) [inline po/es.po]: Version 0.3.0 is already released, please change to 0.3.1.

File: po/es.po [hunk 3]
```diff
@@ -0,0 +1,248 @@
...
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2016-09-23 12:07-0500\n"
+"PO-Revision-Date: 2016-09-23 12:59-0500\n"
+"Last-Translator: Holman Calderón <halecalderon@gmail.com>\n"
+"Language-Team: Spanish; Castilian <>\n"
```
Comment (dengelt) [inline po/es.po]: If you are not part of a language team, leave this blank (i.e. just \n).

File: po/es.po [hunk 4]
```diff
@@ -0,0 +1,248 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the gnome-twitch package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
+# Holman Calderón <halecalderon@gmail.com>, 2016.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: gnome-twitch 0.3.0\n"
+"Report-Msgid-Bugs-To: \n"
```
Comment (dengelt) [inline po/es.po]: Put the bugtracker URL here (see other *.po files)

File: po/es.po [hunk 5]
```diff
@@ -0,0 +1,248 @@
...
+"POT-Creation-Date: 2016-09-23 12:07-0500\n"
+"PO-Revision-Date: 2016-09-23 12:59-0500\n"
+"Last-Translator: Holman Calderón <halecalderon@gmail.com>\n"
+"Language-Team: Spanish; Castilian <>\n"
+"Language: \n"
```
Comment (dengelt) [inline po/es.po]: This field should contain the language code (in this case "es").

**Comment by AlejandroC90** [comment]
Pretty strange, I created the fork from the original proyect the cloned it and the pot file generated is exactly like that and it's outdated, I cloned https://github.com/vinszent/gnome-twitch but I'm having troubles installing the dependence packages, so many dependence to download and to install
is there a way to make it more efficiently?

**Comment by AlejandroC90** [comment]
Well I managed to get all the libraries, but now appears 
image
this is new for me, what should I do? did I miss something while compiling?

**Comment by dengelt** [comment]
You need to build at least one player backend. There are currently 4 available to choose from. See the "Install extra player backends" section in the readme.

**Comment by vinszent** [comment]
Yeah just like @dengelt mentioned, you can just install one locally. Sorry for the troubles, I'm still figuring a lot of this stuff as I go along and it probably won't settle until 1.0 (first stable release) is done.

**Comment by AlejandroC90** [comment]
image
Well, the translation works but I cant manage to get one player backend working, I tried installing from source and installing extra players backend but nothing appears.


image

**Comment by vinszent** [comment]
Can you run ` meson --prefix /usr --libdir lib -Ddo-post-install=true -Dwith-player-gstreamer-clutter=true -Db_lundef=false ..` from the root build directory (` gnome-twitch/build` ).

**Comment by AlejandroC90** [comment]
It seems that everything goes fine, but, 0 backends in gnome-twitch 
image

**Comment by vinszent** [comment]
Wait a second... Have you actually clicked the link in the picture? You can then select a player backend from the ones you have installed.

**Comment by AlejandroC90** [comment]
Yeah, empty.
image

**Comment by vinszent** [comment]
Spanish translation has been merged in from Zanata.

**Pull request closed without merging**